docs/Acceptable-Casks.md

```diff
@@ -67,7 +67,7 @@ for details.
 
 ## Apps that bundle malware
 
-Unfortunately, in the world of software there are bad actors that bundle malware with their apps. Even so, Homebrew Cask has long decided it will not be an active gatekeeper ([macOS already has one](https://support.apple.com/en-us/HT202491)) and [users are expected to know about the software they are installing](#homebrew-cask-is-not-a-discoverability-service). This means we will not always remove casks that link to these apps, in part because there is no clear line between useful app, potentially unwanted program, and the different shades of malware — what is useful to one user may be seen as malicious by another.
+Unfortunately, in the world of software there are bad actors that bundle malware with their apps. Even so, Homebrew Cask has long decided it will not be an active gatekeeper ([macOS already has one](https://support.apple.com/en-us/HT202491)) and users are expected to know about the [software](#homebrew-cask-is-not-a-discoverability-service) they are installing. This means we will not always remove casks that link to these apps, in part because there is no clear line between useful app, potentially unwanted program, and the different shades of malware — what is useful to one user may be seen as malicious by another.
```

Comment:
I would suggest that the previous link was more descriptive than this change. 
The link `#homebrew-cask-is-not-a-discoverability-service` does not speak into the word "software" alone, but contextually does speak into "users are expected to know about the software they are installing".